app/controllers/packages_controller.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
+VENDOR_ID = '123355'
+
 class PackagesController < ApplicationController
   before_action :set_package, only: %i[show update customer_resources]
 
-  deserializable_resource :package, only: :update,
+  deserializable_resource :package, only: %i[update create],
```

Comment:
I am really being nitpicky here but can we please have the create before the update?